windows/internal/vc_install_helper.bat

```diff
@@ -1,23 +1,7 @@
-set VC_VERSION=
-if "%VC_YEAR%" == "2017" if "%CUDA_VERSION%" == "92" (
-    set VC_VERSION=14.13
-    if not "%PYTORCH_BUILD_VERSION:1.6.=%" == "%PYTORCH_BUILD_VERSION%" if "%PYTORCH_BUILD_VERSION:dev=%" == "%PYTORCH_BUILD_VERSION%" (
-        set VC_VERSION=14.11
-    )
-)
-if "%VC_YEAR%" == "2019" set VC_VERSION=14.28.29333
-if not "%VC_VERSION%" == "" (
-    set VSDEVCMD_ARGS=-vcvars_ver=%VC_VERSION%
-)
-if "%VC_YEAR%" == "2017" powershell windows/internal/vs2017_install.ps1 %VC_VERSION%
-if "%VC_YEAR%" == "2019" powershell windows/internal/vs2019_install.ps1 %VC_VERSION%
```

Comment:
Why did you remove this line?

Reply:
1. I don't find where is using vc_version for VS2019 now.
2. Do we still need VS 2017?

at first, I only wanted to change 
`if "%VC_YEAR%" == "2019" powershell windows/internal/vs2019_install.ps1 %VC_VERSION%`
to 
`if "%VC_YEAR%" == "2019" powershell windows/internal/vs2019_install.ps1`

Reply:
The reason is that it's very easy to scrape the lastest VS version but I don't find a way to scrape the matched vc toolchain version in web.

Reply:
I mean if you delete this line, then `vs2019_install.ps1` is not executed, is this the correct behaviour?